content/en-US/api/Screens.md

```diff
@@ -0,0 +1,69 @@
+---
+recipe: api-interface
+title: 'Screens'
+mdn_url: /en-US/docs/Web/API/Screens
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.Screens
+---
+
+## Description
+
+
+The `Screens` interface of the Window Placement API provides multi-screen information and change events.
+
+## Constructor
+
+None.
+
+## Properties
+
+**`Screens.screens`**
+
+Returns an array of `ScreenAdvanced` objects, which describe each connected screen.
+
+**`Screens.currentScreen`**
+
+Returns a `ScreenAdvanced` object for the current screen.
+
+## Eventhandlers
+
+**`Screens.onchange`**
+
+Called when `screens` or `currentScreen` changes.
+
+## Examples
+
+### Logging the number of connected screens
+
+The following example shows how to count the number of connected screens,
+or log a warning if the permission is denied. The `getScreens()` call
+may show a permission prompt if necessary.
+
+```js
+if (screen.isExtended) {
+  window.getScreens().then(
+    screens => {
+      console.log("Screens: " + screens.screens.length);
+    },
+    error => {
+      console.warn("Permission denied");
+    }
+  );
+}
+```
+
+### Detecting when screen properties change
```

Comment:
Actually, this is slightly incorrect. We're planning to add two change events:
[Screen.change](https://github.com/webscreens/window-placement/blob/master/EXPLAINER.md#add-screenchange-an-event-fired-when-screen-attributes-change): an event fired when Screen attributes change
[Screens.change](https://github.com/webscreens/window-placement/blob/master/EXPLAINER.md#add-screens-and-screenadvanced-interfaces-for-additional-screen-info): An event fired when 'screens' or 'currentScreen' changes. (NOTE: Does not fire on changes to attributes of individual Screens)

So the Screen event fires when that particular Screen[Advanced]'s attributes change. The other fires when the Screen.screens frozen array or the Screens.currentScreen object itself changes (i.e. a Screen is connected or disconnected, or the window has moved to another screen).

I hope that's clear! Lmk if I can help clarify or change the proposal.

Reply:
Thanks - clarified! I added another example for detecting currentScreen changes, too.